priv/templates/phx.gen.docker/Dockerfile.eex

```diff
@@ -0,0 +1,89 @@
+# Find eligible builder and runner images on Docker Hub. We use Ubuntu/Debian instead of
+# Alpine to avoid DNS resolution issues in production.
+#
+# https://hub.docker.com/r/hexpm/elixir/tags?page=1&name=ubuntu
+# https://hub.docker.com/_/ubuntu?tab=tags
+#
+#
+# This file is based on these images:
+#
+#   - https://hub.docker.com/r/hexpm/elixir/tags - for the build image
+#   - https://hub.docker.com/_/debian?tab=tags&page=1&name=bullseye-20210902-slim - for the release image
+#   - https://pkgs.org/ - resource for finding needed packages
+#   - Ex: hexpm/elixir:1.12.3-erlang-24.1.4-debian-bullseye-20210902-slim
+#
+ARG BUILDER_IMAGE="hexpm/elixir:1.12.3-erlang-24.1.4-debian-bullseye-20210902-slim"
```

Comment:
Should we leverage:

`Application.spec(:elixir, :vsn)` and `:erlang.system_info(:version)` (not sure how to get the minor version here)

So that the build image matches the elixir version in use?

This might have a downside as we can't ensure the image exists vs using a hard-coded string.

Reply:
Good call. We can do `-erlang-24.1-`, which perhaps is the safest choice.